examples/domainmodel/example/datatypes.dmodel

```diff
@@ -2,6 +2,10 @@ datatype String
 datatype Int
 datatype Decimal
 
+@aaa@wer
+@bbb
+ccc@
```

Comment:
Please remove this from the domainmodel example, now that we've seen that it works.